explainer.md

```diff
@@ -48,35 +48,39 @@ The specification also provides an option for sites to provide a GPC Support Res
 
 The GPC signal is either on or not present. If it is on, then an individual is expressing a privacy choice, for example, to opt out of the sale and data sharing per the California Consumer Privacy Act (CCPA). Sites may choose to support this request beyond what they are legally required to do and their vendors may choose to do so as well.
 
-Requests from a device or browser that supports GPC and where people have activated the GPC signal must contain **both** the header and an addition to the navigator object readable by JavaScript.
+If someone expresses a preference for their information to not be sold or shared, a device or browser that supports the feature will enable GPC signals.  GPC is signaled with an HTTP header and a property that can be read by JavaScript.  Sites can read either signal.
 
 ### 3.1 Header
 
-The following header **must** be on every request:
+When GPC is enabled, a browser includes the following header field in all requests that it makes:
 
-`Sec-GPC: 1`
+```http-message
+Sec-GPC: 1
+```
 
-### 3.2 Navigator Object
+This signal will be absent when no preference has been expressed or where GPC has been disabled.
 
-The property that is added to the navigator object is
+### 3.2 Navigator Object
 
-`navigator.globalPrivacyControl`
+Browsers that support JavaScript will expose `navigator.globalPrivacyControl`.  If `navigator.globalPrivacyControl` is `true`, then GPC has been enabled.
 
-Its value will always be `true` or the property will not be set. **The signal is expected to be active at the time the page loads.**
+The `navigator.globalPrivacyControl` attribute will be present and have a value of `false` if the browser supports GPC but there has either been no preference expressed or GPC has been disabled.  This attribute will be absent only if the browser does not support GPC.
```

Comment:
Two things: 

1. I think it is important in this explainer to note that the `navigator` signal is should be set when the page loads since some extensions might have the option on this. 

2. Is it a concern that the `navigator` property is a behavior inconsistent with the header? I had assumed that it would mirror the header in being true or not present to avoid fingerprinting issues and maintain consistency, but I see the spec has a bool for that property, so if we want that I think the spec needs to change? Opening an issue on this questions.

Reply:
1. I don't think that the explainer needs to say this.  The spec has to, but the explainer is not a document for people who seek to implement, just for those who seek to understand it.  (The implication for an extension developer is clear, however.)
2. I don't see this as an issue.  I'll provide more detail on the issue.

Reply:
After some discussion in PrivacyCG discussing this a little more, I withdraw these objections. Thanks for this change @martinthomson and for surfacing the issue!